devel-tools/update-changes

```diff
@@ -227,12 +225,9 @@ function get_last_rev
 function check_release_tag
 {
     if [ "$release" != "" ]; then
-        git tag -d $release_tag 2>/dev/null
-        git tag -a $release_tag -m "Current stable release."
-        sleep 2 # Make sure git describe picks the next one.
         git tag -d $release 2>/dev/null
```

Comment:
I always found it scary that this script silently deletes my tags (which might even be gone from remotes), so I think this looks great. You could even go so far as to drop the automatic deletion of any previous tag here and below in `check_beta_tag` (if the previous tag already exists the user can still delete it explicitly).

Reply:
Given that the `-B` and `-R` arguments are only used during release management, I'm ok with leaving them as they are as you should be aware of what tags you're moving around at that point.